src/components/settings/Users/EditUser.vue

```diff
@@ -0,0 +1,205 @@
+                        :placeholder="$t('Current Password')"
+                        class="form-control mt-2"
+                        required
+                        :disabled="savingPassword"
+                    />
+                </label>
+            </div>
+
+            <div class="mb-3">
+                <label class="form-label d-block">
+                    {{ $t("New Password") }}
+                    <input
+                        v-model="newPassword"
+                        type="password"
+                        :placeholder="$t('New Password')"
+                        class="form-control mt-2"
+                        required
+                        :disabled="savingPassword"
+                    />
+                </label>
+            </div>
+
+            <div class="mb-3">
+                <label class="form-label d-block">
+                    {{ $t("Repeat New Password") }}
+                    <input
+                        v-model="repeatNewPassword"
+                        type="password"
+                        :placeholder="$t('Repeat New Password')"
+                        class="form-control mt-2"
+                        required
+                        :disabled="savingPassword"
+                    />
+                </label>
+            </div>
+
+            <button class="btn btn-primary" type="submit" :disabled="savingPassword">
+                <span v-show="savingPassword" class="spinner-border spinner-border-sm" role="status" aria-hidden="true" />
+                {{ $t("Update Password") }}
+            </button>
+        </form>
+
+        <h5 class="mt-5 mb-4 settings-subheading">{{ $t("Permissions") }}</h5>
+        <div class="form-check form-switch">
+            <label class="form-check-label">
+                <input
+                    :checked="active"
+                    class="form-check-input"
+                    style="scale: 1.4; cursor: pointer;"
+                    type="checkbox"
+                    :disabled="saving"
+                    @click="debounce(() => { active = !active; save({ active }); })"
+                >
+                <div class="ps-2">{{ $t("Active") }}</div>
+            </label>
+        </div>
+    </template>
+</template>
+
+<script>
+import { Debounce } from "../../../util-frontend.js";
+import { useToast } from "vue-toastification";
+const toast = useToast();
+
+export default {
+    props: {
+        id: {
+            type: String,
+            required: true
+        }
+    },
+
+    data: () => ({
+        loading: false,
+        username: "",
+        saving: false,
+        currentPassword: "",
+        newPassword: "",
+        repeatNewPassword: "",
+        savingPassword: false,
+        active: false
+    }),
+
+    mounted() {
+        this.getUser();
+    },
+
+    methods: {
+        // Used to ignore one of the two "click" events fired when clicking on the checkbox label
+        debounce: new Debounce(),
```

Comment:
Given that this function can only debounce one usage per method, I think renaming this `debounceDoubleClick` would be a better naming

Reply:
renamed `debounceCheckboxClick`